### ISSUE TITLE
Chat can impede the scene content, which is problematic when viewing captions in a scene

### ISSUE THREAD

**Issue by misslivirose** [mozilla/hubs#2392]
**Description**
In the absence of chat scrollback [#1619], the current chat conversation will take up the entire page if many messages are sent in rapid succession. This is distracting and can block critical content within the scene, which is especially important when there is a captioning system being displayed into a room.

![image](https://user-images.githubusercontent.com/4313320/79807620-57981c00-8320-11ea-89c0-d052d3fd476f.png)

Currently, the only way to resolve this is by hiding the UI with the ~ key, but this makes the chat functionality impossible to use as well.

**Expected behavior**
The chat should be contained to a predictable part of the window that does not obscure the entire scene. This would allow a user to align their view such that the chat box does not over-extend over the rest of the environment when they are depending on viewing something in the scene.

**Hardware** 
 - Device: [e.g. Desktop, phone, VR headset]
 - OS: [e.g. Windows, iOS, Linux]
 - Browser [e.g. Firefox]

**Additional context**
There are a number of related issues where UI elements obscure parts of the scene or overlap strangely. This should be considered and fixed as part of #2306, probably with the implementation of chat scrollback.

**Comment by brianpeiris** [comment]
Given we have plans for a chat redesign or implementation for chat scrollback, I'm unsure if we should implement this as described in "Expected behavior", or if we should come up with an intermediate solution (for example, cutting off the messages past a certain height above the chat input).

**Comment by brianpeiris** [comment]
Had a brief discussion about this in today's meeting and it sounded like we were okay with doing some kind of simple, intermediate fix until this is redesigned. We could cut the messages off, add a button to collapse and expand them, or even move the chat/messages off to the side.

**Comment by TechThomas** [comment]
I just want to add this is primarily disconcerting when you are the presenter of the content. It makes it hard to interact with the next/previous slides for your presentation when the chat messages go over them.  I notice at https://meetup.com/a11yvr that presenters pause an extra long time when chat messages are filling up their view.

**Comment by aychtang** [comment]
@brianpeiris @misslivirose: Would also be happy to take this one and implement the simple fix of cutting the the least recent messages off after a certain height.
